beluga_demo_nav2/launch/gazebo_launch.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2024 Ekumen, Inc.
```

Comment:
@Alondruck isn't this launch file already distributed by upstream packages?

Reply:
Just as I wrote it there, no, I think it's better organized the way I did it here compared to how it's set up in Nav2. Additionally, my idea for this package is to include everything necessary to run the tutorial. I also didn't want to depend on another package like beluga-demo, since we agreed that we needed to restructure this repo.